TFEngine.py

```diff
@@ -1344,48 +1345,84 @@ def _do_save(self):
         return False
     return True
 
-  def eval_model(self, output_file=None, output_per_seq_file=None):
+  def eval_model(self, output_file=None, output_per_seq_file=None, loss_name=None, output_per_seq_format="seq_len;score;pos_score"):
     """
     Eval the current model on the eval datasets (dev + eval, whatever is set).
     See also :func:`self.search` for performing beam search.
 
     :param str|None output_file: if given, will save the results to this file (total err/score for each dataset)
     :param str|None output_per_seq_file: if given, will save the err/score for each sequence
+    :param [str]|None loss_name=None: Needed if output_file is given.
```

Comment:
This is not the proper documentation syntax. `[str]` is not allowed (and also not clear what this should mean). `=None` is also not allowed, and also redundant.
Also, the description itself (after `: `) is not really clear to me. I don't really know what this is, and why it's needed.